### ISSUE TITLE
Is this really necessary? `gem 'docker-api', '= 1.34.0'`

### ISSUE THREAD

**Issue by haidangwa** [chef-cookbooks/docker#1015]
### Cookbook version
4.4.1

### Chef-client version
13.10.0

### Platform Details
CentOS 7

### Scenario:
We have a wrapper cookbook that depends on this docker cookbook.  Our systems are installing docker-api 1.34.2 for other reasons, and this cookbook throws warnings about the version. version 1.34.0 is explicitly called out in the metadata.rb.

### Steps to Reproduce:
- Wrap docker 4.4.1 in another cookbook
- system installs docker-api 1.34.2 in another recipe

### Expected Result:
I'm not sure why this cookbook is required to pin only 1.34.0. 1.34.2 should be acceptable.

### Actual Result:
The chef run outputs this warning:
```
[2018-08-01T21:18:21+00:00] WARN: no version of python found, please install Agent version 5.x or higher.
.No matching version found for 'docker-api' in your gem environment.
 - if you are using ChefDK, run the following command: "chef gem install docker-api --version '= 1.34.0'"
 - if you are using bundler, append "gem 'docker-api', '= 1.34.0'" to your Gemfile and run "bundle install"
 - otherwise run: "gem install docker-api --version '= 1.34.0'"
```

**Comment by haidangwa** [comment]
I believe docker-api 1.34.2 is being installed by the inspec gem. Inspec 2.2.50 has a dependency on `train (~> 1.4, >= 1.4.15)`, which in turn depends on `docker-api (~> 1.26)`. This throws a warning when the docker cookbook tries to use docker-api `= 1.34.0`.

**Comment by tas50** [comment]
Fixed by 388193f3ab5cef2e2afb9c616a2e59afa084af93. Thanks for the report
